opentracing-spring-cloud/pom.xml

```diff
@@ -51,6 +51,26 @@
     </dependency>
 
     <dependency>
+      <groupId>org.springframework</groupId>
+      <artifactId>spring-jms</artifactId>
```

Comment:
@pavolloffay how to add activemq-starter test?
As the artifact will conflict with artemis, etc ...